src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -2230,6 +2230,20 @@ void GameObject::Respawn()
     }
 }
 
+bool GameObject::HasConditionalInteraction() const
+{
+    if (GetGOInfo()->GetQuestID())
+        return true;
+
+    if (GetGoType() != GAMEOBJECT_TYPE_AURA_GENERATOR && GetGOInfo()->GetConditionID1())
```

Comment:
where does this aura generator come from?

Reply:
I understand that the conditions for this type are to restrict the application of the aura (aura1-cond1, aura2-cond2) (not related to interaction)